pysb/bng.py

```diff
@@ -236,7 +243,7 @@ def read_simulation_results(self):
         # Read concentrations data
         cdat_arr = numpy.loadtxt(self.base_filename + '.cdat', skiprows=1)
         # Read groups data
-        if len(self.model.observables):
+        if self.model and len(self.model.observables):
```

Comment:
So if the model is loaded straight from a .bngl file and simulated, the observable trajectories won't be read in? Or am I reading this wrong?